<launch>
  
  <arg name="camera_name" default="camera"/>
  <arg name="camera_info_url" default="file://$(find dslr_cam)/config/$(arg camera_name)_info.yaml"/>

  <node name="$(arg camera_name)_node" pkg="dslr_cam" type="dslr_cam_node" output="screen">
    <param name="frame_rate" value="60"/>
    <param name="camera_name" value="$(arg camera_name)"/>
    <param name="camera_frame_id" value="$(arg camera_name)"/>
    <param name="device_id" value="/dev/video2"/>
    <!-- <param name="image_width" value="320"/>
    <param name="image_height" value="240"/> -->
    <!-- <param name="image_width" value="640"/>
    <param name="image_height" value="480"/> -->
    <param name="camera_info_url" value="$(arg camera_info_url)"/>
  </node>
</launch>